com.ibm.wala.core/src/com/ibm/wala/ipa/callgraph/propagation/InstanceFieldKey.java

```diff
@@ -18,6 +18,7 @@
  */
 public class InstanceFieldKey extends AbstractFieldPointerKey {
   private final IField field;
+  private Integer hashCode = null;
```

Comment:
Is there any cost to manipulating Integer objects?  If so, we could just stick with int.  Hopefully, there's a guarantee that it's positive (for initialization and comparison).